lib/smart_udap_harmonization_test_kit/metadata.rb

```diff
@@ -0,0 +1,42 @@
+require_relative 'version'
+
+module SMART_UDAP_HarmonizationTestKit
+  class Metadata < Inferno::TestKit
+    id :smart_udap_harmonization
+    title 'SMART-UDAP Harmonization'
```

Comment:
Full title is SMART-UDAP Harmonization Test Kit

See https://inferno.healthit.gov/test-kits/smart-udap-harmonization/